02.calendar/cal.rb

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'date'
+
+opt = OptionParser.new
+params = {}
+
+opt.on('-m [VAL]') {|v| v }
+opt.on('-y [VAL]') {|v| v }
+opt.parse!(ARGV, into: params)
+
+if params[:m] != nil
+  month = params[:m].to_i
+else
+  month = Date.today.strftime('%m').to_i
```

Comment:
- 一旦 `today = Date.today` のように変数にいれておくと、下のyearのときにも使いまわせると思います。
- [Dateのリファレンスマニュアル](https://docs.ruby-lang.org/ja/latest/class/Date.html)を軽く読んでもらって、使えそうなものがあれば使うようにしてみてください。今回であれば `month` メソッドで代用できると思います。 `strftime` でも悪くはないのですが、どうしても人間にとって読みづらいものになるので、複雑なフォーマットで出力する場合以外は `month` メソッドなどを使うほうが読みやすいです。他に `strftime` を使っているところも他のメソッドがないか探してみてください。

Reply:
`strftime`を使用していた箇所は`Date`内の別のメソッドを使用しました
[Use method on Date instead of strftime](https://github.com/kmiki8242/ruby-practices/pull/2/commits/331c6a6d955b1461a44eb406846e9f8a039f16a6)